.github/actions/happy-cleanup/action.yml

```diff
@@ -15,6 +15,10 @@ inputs:
   tfe_token:
     description: A Terraform Enterprise API token to be used with your happy organization
     required: true
+  working_directory:
+    description: "The happy project root"
+    default: "."
+    required: false
```

Comment:
I think we also want to add this var to `install-happy` and pass it through in this action so it knows where to find the `.happy/version.lock`

Reply:
I don't understand why we would need to add it to `install-happy` when the input var works fine in [deploy-happy-stack](https://github.com/chanzuckerberg/github-actions/blob/main/.github/actions/deploy-happy-stack/action.yml#L42-L45)? Which uses install happy in a similar fashion

Reply:
This is probably fine for now since the edu version of the happy cleanup script has the version hard coded in their action. Generally, we should try to grab the project's version so they match using this input: https://github.com/chanzuckerberg/github-actions/blob/main/.github/actions/install-happy/action.yaml#L20C2-L20C21. But maybe we can cut a new PR for that feature.

Reply:
Ref: https://github.com/chanzuckerberg/edu-github-actions/blob/main/.github/actions/clean-happy-rdev/action.yml#L25

Reply:
We are using deploy without having to declare a version lock file. Should we update our callsites?

https://github.com/chanzuckerberg/bento/blob/main/.github/workflows/deploy.yml#L184-L194